src/navigators/RedirectNavigator.ts

```diff
@@ -26,7 +26,7 @@ export class RedirectNavigator implements INavigator, IWindow {
             throw new Error("No url provided");
         }
 
-        window.location[this._redirectMethod!](params.url);
+        window.location[this._redirectMethod || "assign"](params.url);
```

Comment:
This is actually a valid usage of `!`. `_redirectMethod` must be defined by this point since the caller should have used `INavigate#prepare` to obtain an `IWindow` instance with a `#navigate` implementation. I think a better way to get rid of this warning is to refactor the `IWindow` implementation into a `RedirectWindow` that matches the other navigators.

Reply:
The compiler complains because its not initialized in the constructor, which then leads to a null assertion. In theory the prepare method should be called before. Until we refactor, this fix should not harm as at that point this._redirectMethod should have the proper value...